app/src/test/java/com/example/myspacexdemoapp/ui/launches/LaunchesViewModelTest.kt

```diff
@@ -32,17 +33,20 @@ class LaunchesViewModelTest : TestCase() {
 
     @Test
     fun `when launch by id initialized then success is retrieved`() {
-        val mockResponse: Response<GetLaunchesQuery.Data> =
-            mock(Response::class.java) as Response<GetLaunchesQuery.Data>
+        val mockResponse: ApolloResponse<GetLaunchesQuery.Data> =
+            mock(ApolloResponse::class.java) as ApolloResponse<GetLaunchesQuery.Data>
         val mockData =
             mock(GetLaunchesQuery.Data::class.java)
         val mockLaunches = getLaunches()
-        `when`(mockData.launches()).thenReturn(
+        `when`(mockData.launches).thenReturn(
             mockLaunches
         )
-        `when`(mockResponse.data).thenReturn(mockData)
+        val privateField1: Field = ApolloResponse::class.java.getDeclaredField("data")
```

Comment:
To avoid reflection use `ApolloResponse.Builder` to create `ApolloResponse` object.

It will look like ` val mockResponse =  ApolloResponse.Builder<GetLaunchesQuery.Data>(
            operation = GetLaunchesQuery(),
            requestUuid = UUID.randomUUID(),
            data = mockData
        ).build()`